script/nsis/installer.nsi

```diff
@@ -0,0 +1,53 @@
+Name    "Git LFS"
+OutFile "GitLFSInstaller.exe"
+
+
+InstallDir $DESKTOP
+
+VIProductVersion "0.6.0.0"
+VIAddVersionKey ProductName      "Git LFS"
+VIAddVersionKey Comments         "Git LFS tool"
+VIAddVersionKey CompanyName      "GitHub, Inc"
+VIAddVersionKey LegalCopyright   "GitHub, Inc"
+VIAddVersionKey FileDescription  "Git LFS Tool"
+VIAddVersionKey FileVersion      1
+VIAddVersionKey ProductVersion   1
+VIAddVersionKey InternalName     "Git LFS"
+VIAddVersionKey LegalTrademarks  "something, something, darkside"
+VIAddVersionKey OriginalFilename "GitLFSInstaller.exe"
+
+Function .onGUIInit
+  IfFileExists $PROGRAMFILES64\Git\mingw64\bin\git.exe 0 +3
+    StrCpy $INSTDIR "$PROGRAMFILES64\Git\mingw64\bin"
+    Goto exit
+
+  IfFileExists $PROGRAMFILES64\Git\bin\git.exe 0 +3
+    StrCpy $INSTDIR "$PROGRAMFILES64\Git\bin"
+    Goto exit
+
+  IfFileExists $PROGRAMFILES\Git\bin\git.exe 0 exit
+    StrCpy $INSTDIR "$PROGRAMFILES\Git\bin\git.exe"
+
+  exit:
+    SetOutPath $INSTDIR
+FunctionEnd
```

Comment:
Recommend a stage in the installer which confirms this path & explains that it's going to install LFS in it, option to browse if they want.